### DISCUSSION TITLE
no such file "create_win32_portable.bat". help me ,thanks you all

### DISCUSSION THREAD

**Discussion by lijinlong-yevs** [pawelsalawa/sqlitestudio#4568] [Q&A]
when i compilated sqlitestudio ,no finded "create_win32_portable.bat"file

**Comment by pawelsalawa** [comment]
The wiki instruction for Windows is a bit outdated. *.bat files were replaced with *.tcl files in https://github.com/pawelsalawa/sqlitestudio/tree/master/scripts/win32
Therefore you will need Tcl installed to use them.

I wonder why would you need to do all of this by yourself? I mean there are weekly packages created by Github actions: https://github.com/pawelsalawa/sqlitestudio/actions and it happens that the last one was done 15 hours ago.

**Reply by lijinlong-yevs** [reply]
becase i am a rookie,i want to know fundamentals of compiling in sqlitestudio.i guess i maybe learn more konwledge when i understand very clear.At the same time , thanks for your help,thanks!!!
